onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -226,6 +226,35 @@ def run(self, output_names, input_feed, run_options=None):
                 return self._sess.run(output_names, input_feed, run_options)
             raise
 
+    def run_async(self, output_names, input_feed, callback, user_data, run_options=None):
+        """
+        Compute the predictions asynchronously in a separate thread
```

Comment:
Need to mention that this thread is not a Python thread.

Reply:
Done.